esp/src/eclwatch/Graph7Widget.js

```diff
@@ -100,12 +100,16 @@ define([
                 this.graph = new hpccEclWatch.WUGraph()
                     .target(this.id + "GraphContentPane")
                     .baseUrl("")
-                    .render(function(w) {
+                    .render(function (w) {
                         w
                             .wuid(params.Wuid)
                             .graphID(params.GraphName)
                             .subgraphID(params.SubGraphId)
-                            .render()
+                            .render(function(w) {
```

Comment:
Minor: You adopting syntax above in previous .render method? If so, missing space here.